packages/react-image/etc/react-image.api.md

```diff
@@ -20,6 +20,7 @@ export type ImageCommons = {
     fluid?: boolean;
     circular?: boolean;
     rounded?: boolean;
+    shadow?: boolean;
```

Comment:
should we have a vr-test for this prop ?

Reply:
Great point, yes it should and I'll add it. Thank you